src/ssl.c

```diff
@@ -11994,43 +11994,63 @@ int wolfSSL_SESSION_get_master_key_length(const WOLFSSL_SESSION* ses)
 
 #endif /* OPENSSL_EXTRA */
 
-WOLFSSL_ABI
-void wolfSSL_CTX_set_verify(WOLFSSL_CTX* ctx, int mode, VerifyCallback vc)
-{
-    WOLFSSL_ENTER("wolfSSL_CTX_set_verify");
-    if (ctx == NULL)
-        return;
+typedef struct {
+    byte verifyPeer:1;
+    byte verifyNone:1;
+    byte failNoCert:1;
+    byte failNoCertxPSK:1;
+    byte verifyPostHandshake:1;
+} SetVerifyOptions;
 
-    ctx->verifyPeer     = 0;
-    ctx->verifyNone     = 0;
-    ctx->failNoCert     = 0;
-    ctx->failNoCertxPSK = 0;
-#if defined(WOLFSSL_TLS13) && defined(WOLFSSL_POST_HANDSHAKE_AUTH)
-    ctx->verifyPostHandshake = 0;
-#endif
+static SetVerifyOptions set_verify(int mode)
```

Comment:
Rename function to: wolfssl_mode_to_verify_options
Or something like that.

Reply:
Changed to `ModeToVerifyOptions` to keep the static func theme.